.github/scripts/generate-prdoc.py

```diff
@@ -128,11 +129,11 @@ def yaml_multiline_string_presenter(dumper, data):
 # parse_args is also used by cmd/cmd.py
 # if pr_required is False, then --pr is optional, as it can be derived from the PR comment body
 def setup_parser(parser=None, pr_required=True):
-	allowed_audiences = ["runtime_dev", "runtime_user", "node_dev", "node_operator"]
+	allowed_audiences = ["runtime_dev", "runtime_user", "node_dev", "node_operator", "todo"]
 	if parser is None:
 		parser = argparse.ArgumentParser()
 	parser.add_argument("--pr", type=int, required=pr_required, help="The PR number to generate the PrDoc for.")
-	parser.add_argument("--audience", type=str, nargs='*', choices=allowed_audiences, default=["todo"], help="The audience of whom the changes may concern. Example: --audience runtime_dev node_dev")
+	parser.add_argument("--audience", type=str, nargs='*', choices=allowed_audiences, default=["todo"], help="The audience of whom the changes may concern. Allowed values: " + ", ".join(allowed_audiences))
```

Comment:
I think, `Allowed values: " + ", ".join(allowed_audiences)` is redundant.  

First, all this help feels like it's just for the reader of the code, as we aren't calling `parser.print_help()` anywhere; and help doesn't get displayed if the audience was passed wrong.  
If we would call `parser.print_help()`, it would look like this:
```
options:
  -h, --help            show this help message and exit
  --audience [{runtime_dev,runtime_user,node_dev,node_operator,todo} ...]
                        The audience of whom the changes may concern. Allowed values: runtime_dev,
                        runtime_user, node_dev, node_operator, todo
```
So the options would be mentioned twice.

Reply:
```suggestion
	parser.add_argument("--audience", type=str, nargs='*', choices=allowed_audiences, default=["todo"], help="The audience of whom the changes may concern. Example: --audience runtime_dev node_dev")
```